SilentReporter.js

```diff
@@ -32,17 +33,22 @@ class SilentReporter {
     }
 
     if (!testResult.skipped) {
-      if (testResult.failureMessage) {
-        if (this.showPaths) this.stdio.log('\n' + test.path);
-        this.stdio.log('\n' + testResult.failureMessage);
+      const didUpdate = this._globalConfig.updateSnapshot === 'all';
+      const hasUncheckedSnapshots =
+        !didUpdate && testResult.snapshot && testResult.snapshot.unchecked;
+      const hasFailures = testResult.failureMessage || hasUncheckedSnapshots;
+
+      if (this.showPaths && hasFailures) {
+        this.stdio.log('\n' + test.path);
```

Comment:
It should include some text I think, not _just_ the path. ``log(`Obsolete snapshot at ${test.path}`)`` or something?

Reply:
That text will be logged via `helpers.getSnapshotStatus` [L60](https://github.com/rickhanlonii/jest-silent-reporter/pull/19/files#diff-c0b812ff10f8336c66a4e1aabc936fddR60)

Reply:
added result example in the description.

Reply:
ah, perfect!